src/main/java/uk/gov/pay/api/model/ledger/TransactionState.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.pay.api.model.ledger;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class TransactionState {
```

Comment:
Not sure what's the benefit of having `TransactionState` which has been forced to have exactly the same structure as `PaymentState`.
In this case it's just unnecessary duplicate imho.

Reply:
Doesn't feel right to have `PaymentState` in `RefundTransactionFromLedger`.   Given `PaymentState` is within publicapi domain, both of these represent different ends of the journey although the structure is identical.